network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -53,21 +53,21 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
         "wagtailpages.BuyersGuideEditorialContentIndexPage",
     ]
 
-    hero_featured_article = models.ForeignKey(
-        "wagtailpages.BuyersGuideArticlePage",
+    hero_featured_page = models.ForeignKey(
+        "wagtailcore.Page",
```

Comment:
Using the [wagtail docs](https://docs.wagtail.org/en/stable/reference/pages/panels.html#pagechooserpanel) as reference, I updated the FK relationship in **both fields** to point to `wagtailcore.Page`. This allows us to select from more than one page type.


The choices are then limited to `BuyersGuideCampaignPage` and `BuyersGuideArticlePage` in both fields `PageChooserPanel` definition:

https://github.com/MozillaFoundation/foundation.mozilla.org/blob/4f75c5fd496d56e533157b38061ff08e07a03135/network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py#L102-L105

Reply:
Thanks for the explanation and the reference to the docs 👍